rfcs/text/macho/macho.yml

```diff
@@ -0,0 +1,173 @@
+              description: Size of load commands of the Mach-O header.
+              type: long
+              level: extended
+              format: bytes
+              example: 3888
+
+            - name: type
+              description: Type of the load commands for the Mach-O header.
+              type: keyword
+              level: extended
+              example: LC_SYMTAB, 0x2c
+
+        - name: magic
+          description: Magic field of the Mach-O header.
+          type: keyword
+          level: extended
+          example: 0xfeedfacf
+
+        - name: flags
+          description: Flags set in the Mach-O header.
+          type: keyword
+          level: extended
+          example: TWOLEVEL, 0x4000000
+
+    - name: segments
+      level: extended
+        description: Segment information for the file.
+      type: object
+      fields:
+        - name: vmaddr
+          description: Memory address of this segment.
+          type: keyword
+          level: extended
+          example: 0x0
+
+        - name: name
+          description: Name of this segment.
+          type: keyword
+          level: extended
+          example: __TEXT, __DATA, __IMPORT
+
+        - name: vmsize
+          description: Memory size of this segment.
+          type: keyword
+          level: extended
+          example: 0x4c000
+
+        - name: fileoff
+          description: File offset of this segment.
+          type: keyword
+          level: extended
+          example: 0x0
+
+        - name: filesize
+          description: Amount of memory to map from the file.
+          type: keyword
+          level: extended
+          example: 0x4c000
+
+        - name: sections
+          level: extended
+          description: Section information for the segment of the file.
+          type: flattened
+          fields:
+            - name: flags
+              description: Section flags for the segment of the file.
+              type: keyword
+              level: extended
+              example: SECTION_ATTRIBUTES_USR, S_8BYTE_LITERALS
+
+            - name: name
+              description: Section name for the segment of the file.
+              type: keyword
+              level: extended
+              example: __objc_classname, __stub_helper
+
+            - name: type
+              description: Section type for the segment of the file.
+              type: keyword
+              level: extended
+              example: S_REGULAR, S_CSTRING_LITERALS
+
+        - name: offset
+          description: Offset of the segment.
+          type: long
+          format: bytes
+          level: extended
+          example: 0
+
+        - name: size
+          description: Segment limit size.
+          type: long
+          format: bytes
+          level: extended
+          example: 123456
+
+        - name: flags
+          description: Segment flags.
+          type: keyword
+          level: extended
+          example: 0x0
+
+    - name: cdhash
```

Comment:
this looks like it's missing from the RFC document itself.

Reply:
Fix't